donkeycar/tests/test_train.py

```diff
@@ -1,225 +0,0 @@
-# -*- coding: utf-8 -*- #
-import pytest
-import os
-
-from donkeycar.templates.train import multi_train
-from donkeycar.parts.datastore import Tub
-from donkeycar.parts.simulation import SquareBoxCamera, MovingSquareTelemetry
-
-from donkeycar.templates.train import gather_records, collate_records
-
-#fixtures
-from .setup import tub, tub_path, on_pi
-from .setup import create_sample_tub
-
-def cfg_defaults(cfg):
-    cfg.MAX_EPOCHS = 1
-    cfg.BATCH_SIZE = 10
-    cfg.SHOW_PLOT = False
-    cfg.VERBOSE_TRAIN = False
-    cfg.OPTIMIZER = "adam"
-    cfg.TARGET_H = cfg.IMAGE_H - cfg.ROI_CROP_TOP - cfg.ROI_CROP_BOTTOM
-    cfg.TARGET_W = cfg.IMAGE_W
-    cfg.TARGET_D = cfg.IMAGE_DEPTH
-    cfg.NUM_LOCATIONS = 10
-
-
-@pytest.mark.skipif(on_pi() == True, reason='Too slow on RPi')
-def test_train_cat(tub, tub_path):
-    t = Tub(tub_path)
-    assert t is not None
-
-    import donkeycar.templates.cfg_complete as cfg
-    tempfolder = tub_path[:-3]
-    model_path = os.path.join(tempfolder, 'test.h5')
-    cfg_defaults(cfg)
-
-    tub = tub_path
-    model = model_path
-    transfer = None
-    model_type = "categorical"
-    continuous = False
-    aug = False
-    multi_train(cfg, tub, model, transfer, model_type, continuous, aug)
-
-
-@pytest.mark.skipif(on_pi() == True, reason='Too slow on RPi')
-def test_train_linear(tub, tub_path):
```

Comment:
I think we should keep some training tests.

Reply:
These don't test the new training setup.